src/module.ts

```diff
@@ -30,6 +41,7 @@ export default defineNuxtModule<ModuleOptions>({
     applicationId: '',
     apiKey: '',
     lite: true,
+    instantSearch: { theme: 'satellite' },
```

Comment:
I would make it disabled by default. Similarly to crawler and lite, let's focus on delivering the lightest setup possible and if a user needs that, then he can enable additional functionalities.

Reply:
By disabled I mean `false`

Reply:
I agree, I'll change that